noble/assetlist.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "noble",
+  "assets": [
+    {
+      "description": "The permissioned staking asset for Noble Chain",
+      "denom_units": [
+        {
+          "denom": "ustake",
+          "exponent": 0,
+          "aliases": [
+            "microstake"
+          ]
+        },
+        {
+          "denom": "stake",
+          "exponent": 6
+        }
+      ],
+      "base": "ustake",
+      "name": "Stake",
+      "display": "stake",
+      "symbol": "STAKE"
+    },
+    {
+      "description": "Frienzies are an IBC token redeemable exclusively for a physical asset issued by the Noble entity.",
+      "denom_units": [
+        {
+          "denom": "ufrienzies",
+          "exponent": 0,
+          "aliases": [
+            "microfrienzies"
+          ]
+        },
+        {
+          "denom": "frienzies",
+          "exponent": 6
+        }
+      ],
+      "base": "ufrienzies",
+      "display": "ufrienzies",
+      "name": "frienzies",
+      "symbol": "FRNZ"
+    },
+    {
+      "description": "USD Coin",
```

Comment:
We have the image for USDC under _non-cosmos/ethereum/usdc.svg, which you may reference from here (or re-upload the image in this folder). And I'd recommend logo images (SVG preferred) for the other assets, too

Reply:
@Jelena647 do you want this to be the `USDC` Logo, the `noble` Logo, or do you have one for `FRNZ`  ?

Reply:
I'm going to suggest this be approved for now, and will confirm with @Jelena647 if we want to change.

https://github.com/strangelove-ventures/chain-registry/commit/0ce0a29d8bcdf87241f1257839cfc1191a9dc1a0

Reply:
Ok, please place the image into an /images/ folder (within the /noble/ folder) , and then you can refer to that image using
```
"logo_URIs": {
  "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/frnz.png"
}
```